image/descriptor.go

```diff
@@ -32,6 +33,10 @@ type descriptor struct {
 	Size      int64  `json:"size"`
 }
 
+func (d *descriptor) normalizeDigest() string {
+	return strings.Replace(d.Digest, ":", "-", -1)
+}
```

Comment:
On Fri, Jul 22, 2016 at 07:52:32AM -0700, Antonio Murdaca wrote:
> I believe this should go in asap. Potentially #159 can take a lot
> more time to be reviewed (not maintainers reviewed it yet)

I think it would be more efficient to put better abstractions into
place as soon as possible, instead of iterating on an approach we
expect to replace.  But that's a throughput vs. latency issue for the
maintainers to figure out ;).